give.spec

```diff
@@ -1,8 +1,9 @@
-#don't strip binaries
+# The default behavior of rpmbuild is to strip binaries after installation
+# in order to reduce file size. The two macros defined below override that
+# behavior and compresses the man and info pages for debugging purposes.
```

Comment:
Good. Note that the default behavior depends on the OS distribution your rpm is building on. Avoid generalization here.

Provide the URL link you used to find this information; others may also find it useful in the future.